packages/react-native-aztec/package.json

```
@@ -1,29 +1,42 @@
 {
   "name": "@wordpress/react-native-aztec",
   "version": "0.1.11",
+  "description": "Aztec view for react-native.",
   "private": true,
-  "license": "GPL-2.0",
+  "author": "The WordPress Contributors",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "wordpress",
+    "react-native"
+  ],
+  "homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/react-native-aztec/README.md",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/WordPress/gutenberg.git",
+    "directory": "packages/react-native-aztec"
+  },
+  "bugs": {
+    "url": "https://github.com/WordPress/gutenberg/issues"
+  },
+  "dependencies": {
+    "prop-types": "15.6.0"
```

For packages, the preferred approach is to use ranges for dependencies:
```suggestion
    "prop-types": "^15.6.0"
```